CHANGELOG.md

```diff
@@ -68,17 +68,29 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Middleware support (for e.g. for centrally handling authorization).
 
 ## [1.1.0] - 2019-05-30
+### Added
+- Travis builds for tags.
 
 ## [1.0.6] - 2019-05-19
+### Added
+- Custom logger per route support.
+
+### Fixed
+- Fixed buffer reading when timeout or disconnect occurs.
+- Fixed cookie formatting: stop url-encoding cookie path and quoting cookie expire date.
 
 ## [1.0.5] - 2018-09-03
+### Changed
+- Protocol upgrade: detaching mechanism is re-worked.
 
 ## [1.0.4] - 2018-08-31
```

Comment:
Looks correct.
Changes in 1.0.4: https://github.com/tarantool/http/compare/1.0.3...1.0.4